feat: rebranding docs (#1856)
PrestaShop/docs: development/ui-kit/how-to-use-it.md

---

> title: How to use it ?

Reviewer: ```suggestion
title: How to use it?
```

weight: 63
---

> # How to use new set of CSS variables ?

Reviewer: ```suggestion
# How to use a new set of CSS variables?
```

> To maintain consistency across the PrestaShop back office, CSS variables have been introduced in `_root.scss`. This guide explains how to use these variables in PrestaShop.

Reviewer: ```suggestion
To maintain consistency across the PrestaShop back office, CSS variables have been introduced in `_root.scss`. This guide explains how to use them.
```

> ## Using CSS variables with v9 and higher

Reviewer: ```suggestion
## Using CSS variables in PrestaShop 9.0+
```

> In PrestaShop v9 and higher, use the CSS variables introduced in `_root.scss` for consistent styling in the Back Office.

Reviewer: ```suggestion
In PrestaShop v9 and later, we use the CSS variables introduced in `_root.scss` for consistent styling in the back office.
```

### Usage Example

```scss
/* CSS: Apply dark background on a block */
.my-block {
  background-color: var(--cdk-primary-800);
}

/* SCSS: Apply dark background on a block */
.my-block {
  background-color: var(--#{$cdk}primary-800);
}
```

**Note:**

- `--cdk-primary-800` is a CSS variable defined in `_root.scss`. It ensures consistent color application across the back office.

> - Refer to `_root.scss` for the latest <a href="https://github.com/PrestaShop/prestashop-ui-kit/blob/develop/scss/_root.scss" target="_blank">list of available variables</a>.

Reviewer: Here I think you should use `refrel` to the docs page you created, right?
